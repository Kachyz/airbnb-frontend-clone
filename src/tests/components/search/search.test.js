import React from 'react'
import { shallow } from 'enzyme'
import SearchFrom from "../../../components/search/search";

test('should render SearchFrom page correctly', () => {
  const wrapper = shallow(<SearchFrom />)

  wrapper.find('TextField').at(0).simulate('change',{
    target: {
      value: '42'
    }
  })

  expect(wrapper.state('search')).toEqual("42")

  expect(wrapper).toMatchSnapshot()
})