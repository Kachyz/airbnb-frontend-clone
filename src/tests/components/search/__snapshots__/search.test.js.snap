// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchFrom page correctly 1`] = `
<div>
  <TextField
    id="search"
    onChange={[Function]}
    required={false}
    select={false}
    value="42"
  />
  <WithStyles(Button)
    color="secondary"
    onClick={[Function]}
    variant="contained"
  >
    Search
  </WithStyles(Button)>
</div>
`;
